tests/func/test_gdrive.py

```diff
@@ -0,0 +1,68 @@
+from subprocess import check_call
+import os
+import tempfile
+
+from dvc.main import main
+from dvc.remote.gdrive import RemoteGDrive
+from dvc.remote.gdrive.oauth2 import OAuth2
+
+from tests.basic_env import TestDvc
+
+
+class TestGDrive(TestDvc):
+    def test(self):
+        main(["add", self.FOO, self.BAR, self.DATA, self.DATA_SUB])
```

Comment:
`main` will return 1 on error, so you'll need to check that. Or, you could use API function instead, that will actually raise an exception (e.g. `self.dvc.add(self.FOO)`. Also, we are using pytest for new tests, so please convert it too.

Reply:
Fixed. Also, I think it is better to use the separate process via `check_call` to don't touch the coverage.

Reply:
Oh, missed the py.test point.

Reply:
Fixed.